src/config/example.toml

```diff
@@ -146,3 +146,4 @@ bloom_false_positive = 0.001
 backup_storage_url = "memory"
 backup_storage_directory = "backup"
 telemetry_enabled = true
+max_concurrent_creating_streaming_jobs = 1
```

Comment:
Is 1 sufficient for production? 👀

Reply:
Well, @fuyufjh suggests to force the creation to be serial by default. I also concern about it, for jobs that don't have a lot of historical data, the resource consumption may not be that high. In this case the concurrency can actually be a little larger. 🥵

Reply:
But setting it to 1 by default is acceptable to me as well. OOM can be effectively avoided. We can add some more obvious and rich descriptions to this config so that users can set higher parallelism for better efficiency.

Reply:
> Yeah. The Well, @fuyufjh suggests to force the creation to be serial by default. I also concern about it, for jobs that don't have a lot of historical data, the resource consumption may not be that high. In this case the concurrency can actually be a little larger. 🥵

Yeah. I think 1 is a reasonable default value because by default every table/MV will have full parallelism to process data, which means in most cases 1 creating MV should utilize the CPU/IO resources.